content/en/serverless/azure_app_services/azure_app_services_windows.md

```diff
@@ -244,6 +244,87 @@ Learn more about [custom metrics][10].
 [9]: /developers/dogstatsd/?tab=java#code
 [10]: /metrics/
 {{% /tab %}}
+{{% tab "Node.js" %}}
+### Requirements
+
+If you haven't already, set up the [Datadog-Azure integration][1] first.
+
+Datadog's Azure App Service Node.js extension supports Azure App Service Web Apps. Function Apps are not supported.
```

Comment:
```suggestion
Datadog's Azure App Service Node.js extension supports Azure App Service Web Apps. Function Apps are not yet supported.
```

Reply:
As a matter of documentation policy, we do not use the phrasing "not yet" or anything that suggests the future existence of functionality that is not presently available to customers.